pkg/server/routes.go

```diff
@@ -108,6 +110,7 @@ func (s *Server) routes(
 		FetchSystemIntakeByID: services.NewFetchSystemIntakeByID(
 			serviceConfig,
 			store.FetchSystemIntakeByID,
+			func(ctx context.Context, intake *models.SystemIntake) (bool, error) { return true, nil },
```

Comment:
I'd like to understand why we're not just defining these somewhere else. I think it'll follow the other patterns and make it easier to drop in the proper authorization when we get to that.